src/sidebar/app.js

```diff
@@ -5,7 +5,14 @@ require('../shared/polyfills');
 
 var raven;
 
-var settings = require('../shared/settings')(document);
+// Read settings rendered into sidebar app HTML by service/extension.
+var settingsReader = require('../shared/settings');
+var settings = settingsReader(document);
+
+// FIXME: Remove this once the service and extension have been changed to
+// render settings to script tags with a 'js-hypothesis-config' class.
+Object.assign(settings, settingsReader(document, 'js-hypothesis-settings'));
```

Comment:
Would it not make more sense to simply use both classnames in the service until this fix is fully rolled out?